packages/fluentui/react-northstar/src/components/Carousel/utils.ts

```diff
@@ -0,0 +1,23 @@
+export function getAnimationName({
+  active,
+  dir,
+  animationEnterFromPrev,
+  animationEnterFromNext,
+  animationExitToPrev,
+  animationExitToNext,
+}) {
+  const initialMounting = typeof dir === 'undefined';
+
+  let animationName = '';
+  if (!initialMounting) {
+    if (!active) {
+      animationName = dir === 'start' ? animationExitToPrev : animationExitToNext;
+    }
+
+    if (active) {
+      animationName = dir === 'start' ? animationEnterFromNext : animationEnterFromPrev;
+    }
+  }
+
+  return animationName;
```

Comment:
```suggestion

  if (!initialMounting) {
    if (!active) {
      return dir === 'start' ? animationExitToPrev : animationExitToNext;
    }

    if (active) {
      return = dir === 'start' ? animationEnterFromNext : animationEnterFromPrev;
    }
  }

  return '';
```

nit: Can we use early return? It's more readable IMO.